pkg/tfgen/docs.go

```diff
@@ -162,7 +162,7 @@ func getDocsForProvider(g *generator, org string, provider string, resourcePrefi
 
 	doc, err := parseTFMarkdown(g, info, kind, string(markdownBytes), markdownFileName, resourcePrefix, rawname)
 	if err != nil {
-		return parsedDoc{}, nil
+		return parsedDoc{}, err
```

Comment:
I don't know why we never returned the error from parsing the markdown here, but this caused me to lose several hours while trying to understand why schema generation wasn't working properly for me. Because this error wasn't being bubbled-up, I missed out the fact that example parsing was failing due to `tf2pulumi` not existing in my `GOPATH` and therefore entire `Description` was being ignored.